examples/tutorial/tutorial_notebooks/introduction/exercise_2.ipynb

```diff
@@ -503,7 +726,7 @@
  ],
```

Comment:
suggestion: instead of a plain text instructing users to move onto the next exercise, consider adding a link to the next exercise directly.

Reply:
@jeffreykennethli I added hyperlink.